### PR TITLE
style(website): integrate new homepage design

### PR DESCRIPTION
**Pull request by ivangabriele**
https://dinum-metiers-num-staging-pr510.osc-fr1.scalingo.io/

### CODE REVIEW

**Comment by ivangabriele** [comment]
[Intégrer la nouvelle UI de la page d'accueil et les offres d'emploi](https://trello.com/c/UtBTMkeC/104-int%C3%A9grer-la-nouvelle-ui-de-la-page-daccueil-et-les-offres-demploi)

File: pages/theme.tsx [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+  th {
+    text-align: left;
+  }
+
+  tbody > tr > th {
+    vertical-align: top;
+  }
+`
+
+export default function ThemePage() {
```
Comment (pybuche) [inline pages/theme.tsx:40]: @ivangabriele What did you want to do with this? Put it in a Storybook at some point?

File: public/images/running-bot.svg:Zone.Identifier [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
```
Comment (pybuche) [inline public/images/running-bot.svg:Zone.Identifier:1]: @ivangabriele I didn't know this kind of file. What's it for?

**Review by pybuche — APPROVED** [review]
LGTM!

**Pull request merged**